tests/integration/README.md

```diff
@@ -0,0 +1,38 @@
+# Building the RPM package
+
+1. Checkout the project from OBS into some directory:
+```
+mkdir -p ~/obs/
+cd ~/obs/
+osc -A https://api.suse.de co Devel:SMT:SLE-12-SMT
+```
+2. Make the tarball in git working directory by running `make package`
+3. Update the data from OBS by the tarball and metadata from git:
+```
+cp package/* ~/obs/Devel:SMT:SLE-12-SMT/smt
```

Comment:
No, there are multiple things other than SMT checked out from OBS (YaST module and some dependencies).